activerecord/test/cases/counter_cache_test.rb

```diff
@@ -148,6 +164,17 @@ class ::SpecialReply < ::Reply
     end
   end
 
+  test "reset counters for cpk model" do
+    order = Cpk::Order.first
+    # throw the count off by 1
+    Cpk::Order.increment_counter(:books_count, order.id)
+
+    # check that it gets reset
+    assert_difference "order.reload.books_count", -1 do
```

Comment:
Are you sure eval is cool here? It might be slow. How about lambda?

```suggestion
    assert_difference ->{ order.reload.books_count }, -1 do
```

Reply:
That is the style used within this file. The slowness won't be noticeable, but I also don't like this pattern, so time to break it.